SystemImageEX/Study_2_self/Controller/SecondViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  SecondViewController.swift
+//  Study_2_self
+//
+//  Created by 이중엽 on 2023/03/02.
+//
+
+import UIKit
+
+protocol SendActionDelegate {
+    func reload()
+}
+
+enum ViewType {
+    case add
+    case update
+}
+
+class SecondViewController: UIViewController {
+    @IBOutlet weak var image: UIImageView!
+    @IBOutlet weak var imageName: UILabel!
+    @IBOutlet weak var inputTextField: UITextField!
+    @IBOutlet weak var checkButton: UIButton!
+    @IBOutlet weak var updateButton: UIButton!
+    
+    var symbol: Symbol?
+    var index: Int?
+    var viewType: ViewType = .add
+    var isCheck = false
+    var delegate: SendActionDelegate?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        inputTextField.delegate = self
+        setUpInitialData()
+    }
+
+    func setUpInitialData() {
+        if viewType == .add {
+            image.image = nil
+            imageName.text = ""
+            updateButton.setTitle("저장", for: .normal)
+        } else {
+            image.image = symbol?.image
+            imageName.text = symbol?.name
+            updateButton.setTitle("수정", for: .normal)
+        }
+        
+        updateButton.isEnabled = false
+    }
+    
+    @IBAction func checkButtonClicked(_ sender: UIButton) {
+        if inputTextField.text == "" {
+            showAlert(title: "빈칸", message: "입력된 값이 없습니다.")
+            return
+        }
+        
+        guard let myImage = UIImage(systemName: inputTextField.text!) else {
+            showAlert(title: "오류", message: "해당 이름으로 된 이미지가 존재하지 않습니다.")
+            return
+        }
+        
+        image.image = myImage
+        imageName.text = inputTextField.text!
+        updateButton.isEnabled = true
+    }
+    
+    @IBAction func saveButtonClicked(_ sender: UIButton) {
+        let newSymbolImage = UIImage(systemName: inputTextField.text!)!
+        let newSymbolName = inputTextField.text!
+        let newSymbol = Symbol(image: newSymbolImage, name: newSymbolName)
+        
+        //중복확인
```

Comment:
코멘트를 작성할 때는 꼭 // 뒤에 공백을 하나 주세요.
이 파일의 모든 코멘트에 대해 적용 부탁드립니다~

```suggestion
        // 중복확인
```

Reply:
@keenkim1202 
수정 완료했습니다.
확인 부탁드립니다.